eos_data_distribution/ndn/dbus/base.py

```diff
@@ -251,7 +251,9 @@ def __init__(self, name, dbus_name, skeleton):
         self._dbus_name = build_dbus_name(dbus_name, name)
         self._interface_skeleton_object = skeleton
 
-        self.con = Gio.bus_get_sync(BUS_TYPE, None)
+        address = Gio.dbus_address_get_for_bus_sync(BUS_TYPE, None)
+        self.con = Gio.DBusConnection.new_for_address_sync(address, Gio.DBusConnectionFlags.NONE, None, None)
```

Comment:
we used to need 2 DBus connections because if i reused the same one to publish on 2 different interfaces everything got exported to both.
this code is not strictly required now, but it's more correct than the former one.